### DISCUSSION TITLE
Output existing environment variables as suggestions when using variables

### DISCUSSION THREAD

**Discussion by erwinmoretz** [thunderclient/thunder-client-support#1264] [Ideas]
Hi,

I have an idea for making the usage of variables in thunder client easier. 

If there is an initiated environment variable, it could be recommended in the text fields if you start writing the curly brackets **{{**. Like an autofill option.

Here is a screenshot where i think this feature would be nice: 

![idea_thunderclient](https://github.com/rangav/thunder-client-support/assets/112586589/6c3c0f3e-f95c-45de-ba92-50032f2b53da)

**Comment by rangav** [comment]
Hi @erwinmoretz thanks for the feedback.

There is existing feature request 
https://github.com/rangav/thunder-client-support/issues/1194
